cli/core/findStalePods.go

```diff
@@ -85,11 +105,17 @@ func isEigenpod(eth *ethclient.Client, chainId uint64, eigenpodAddress string) (
 		return false, fmt.Errorf("PodOwnerShares() failed: %s", err.Error())
 	}
 
+	balance, err := eth.BalanceAt(context.Background(), common.HexToAddress(eigenpodAddress), nil)
+	if err != nil {
+		return false, fmt.Errorf("balance check failed: %s", err.Error())
+	}
+
 	// Simulate fetching from contracts
 	// Implement contract fetching logic here
 	cache.PodOwnerShares[eigenpodAddress] = PodOwnerShare{
-		Shares:     podOwnerShares.Uint64(),
-		IsEigenpod: true,
+		Shares:                podOwnerShares.Uint64(),
+		ExecutionLayerBalance: weiToGwei(balance.Uint64()),
```

Comment:
Note: shares are in wei here, and you're converting the balance to gwei

Reply:
updated everything to be explicitly wei or gwei